### DISCUSSION TITLE
Great to see

### DISCUSSION THREAD

**Discussion by jaymunro** [codyc1515/ha-contact-energy#10] [General]
Thank you Cody! Now I can get an accurate picture of energy usage from the outside to compare with the internal energy usage sensors. 

May I suggest putting a reference to the fact that it is for the NZ Contact Energy company in the description? Not sure if there are other such companies around the world, but I imagine it's a fairly common name.

Are there any other potential sensors available in the Contact API, such as billing, fixed costs vs variable costs, peak, off-peak, free, etc.?

**Comment by hydnandrews** [comment]
Hi Cody,

Thank you for this integration - its been working great in my HA!
Any potential of implementing Gas consumption and Broadband Usage?

Cheers!
